.gitignore

```diff
@@ -115,4 +2142,4 @@ lib/libs2d_engine.a
 !/lib/PR/hvqm/*.o
 
 # .orig files
-*.orig
+*.orig
```

Comment:
```suggestion
*.orig

```
Why was the end of file newline removed here?

Reply:
good call! and I'm not sure, i thought i had auto newline turned on but new vscode, good callouts